jsonpath/src/commonMain/kotlin/at/asitplus/jsonpath/core/functionExtensions/countFunctionExtension.kt

```diff
@@ -0,0 +1,22 @@
+package at.asitplus.jsonpath.core.functionExtensions
+
+import at.asitplus.jsonpath.core.JsonPathFilterExpressionValue
+import at.asitplus.jsonpath.core.JsonPathFilterExpressionType
+import at.asitplus.jsonpath.core.JsonPathFunctionExtension
+import kotlinx.serialization.ExperimentalSerializationApi
+import kotlinx.serialization.json.JsonPrimitive
+
+/**
+ * specification: https://datatracker.ietf.org/doc/rfc9535/
+ * date: 2024-02
+ * section: 2.4.5.  count() Function Extension
+ */
+@OptIn(ExperimentalSerializationApi::class)
+internal val countFunctionExtension by lazy {
```

Comment:
@JesusMcCloud Regarding https://github.com/a-sit-plus/jsonpath/pull/9#issuecomment-2100896623
Is there a better way to define this extension without making it a global variable?